src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(num) {
+      calculator.result += num;
+    },
+    subtract(num) {
+      calculator.result -= num;
+    },
+    multiply(num) {
+      calculator.result *= num;
+    },
+    divide(num) {
+      if (num === 0) {
+        console.error('Division by zero is not allowed');
```

Comment:
Don't need to use console, because now tests are failed
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/throw
```suggestion
        throw new Error('Division by zero is not allowed');
```